.travis.yml

```diff
@@ -1,4 +1,4 @@
-osx_image: xcode10.2
+osx_image: xcode11.1
```

Comment:
Should be using 11.3 now that it has been released by Apple

Reply:
I actually did, originally, but Travis didn't have it available at the time of writing this. Looks like it's available now, so I'll go ahead and update when I get a chance